tools/gen_docs.py

```diff
@@ -128,13 +128,13 @@ def test_hash(verbose=False, debug=False):
     match = result == original
     if not match:
         print("FAIL: Internal documents are outdated! Please update via \"python tools/gen_docs.py\"")
-    return int(not match)
+    return str(not match)
 
 
 if __name__ == "__main__":
-    import argparse
+    import argparse as argp
```

Comment:
This is a purely subjective change. Considering that this variable is used in only one place, we gain nothing by making the variable shorter as no one will have to type it again anyways.